internal/auth/oidc/auth_method.go

```diff
@@ -270,9 +270,10 @@ func (am *AuthMethod) isComplete() error {
 }
 
 type convertedValues struct {
-	Algs  []interface{}
-	Auds  []interface{}
-	Certs []interface{}
+	Algs   []interface{}
+	Auds   []interface{}
+	Certs  []interface{}
+	Scopes []interface{}
```

Comment:
Consider renaming Scopes to ClaimScopes since the name Scope collides with a concept in boundary and the convertedValue struct does not only capture claim values.

Reply:
ty!